Modules/Filtering/Thresholding/include/itkThresholdImageFilter.h

```diff
@@ -103,15 +103,15 @@ class ITK_TEMPLATE_EXPORT ThresholdImageFilter : public InPlaceImageFilter<TImag
   /** Get the "outside" pixel value. */
   itkGetConstMacro(OutsideValue, PixelType);
 
-  /** The values greater than or equal to the value are set to OutsideValue. */
+  /** The values greater than "threshold" are set to OutsideValue. */
```

Comment:
Just a possible suggestion: you might consider using backtick characters  (``), instead of double-quotes, for identifiers like `threshold`. Doxygen does process those  backticks correctly. Anyway, no show-stopper.

Reply:
@blowekamp Thanks for the backticks 😺